tests/meta/compress/test_compress_meta_fastq.py

```diff
@@ -10,14 +13,15 @@ def test_compress_case_fastq_one_sample(populated_compress_fastq_api, sample, ca
     # GIVEN a populated compress api
 
     # WHEN Compressing the bam files for the case
-    res = compress_api.compress_fastq(sample)
+    with mock.patch.object(CompressAPI, "_is_spring_archived", return_value=False):
+        res = compress_api.compress_fastq(sample)
 
```

Comment:
and update usages below
```suggestion
        result = compress_api.compress_fastq(sample)

```